apps/chat/src/components/Promptbar/components/Prompt.tsx

```diff
@@ -469,7 +480,7 @@ export const PromptComponent = ({
             ref={refs.setFloating}
             {...getFloatingProps()}
             className={classNames(
-              'absolute right-3 z-50 flex justify-end group-hover/prompt-item:visible',
+              'absolute right-3 z-50 flex justify-end group-hover:visible',
```

Comment:
Are you sure that hover logic isn't broken?

Reply:
I checked and I don't think it's breaking anything.
BG of `<ShareIcon />` was broken because of the different groups.